core/src/main/resources/lib/form/rowSet.jelly

```diff
@@ -37,19 +37,15 @@ THE SOFTWARE.
     </st:attribute>
   </st:documentation>
 
-  <div class="rowSet-container">
-    <j:choose>
-      <j:when test="${attrs.ref==null and attrs.name==null}">
-        <!-- noop -->
-        <d:invokeBody />
-      </j:when>
-      <j:otherwise>
-        <div ref="${attrs.ref}" class="row-set-start row-group-start tr" style="display:none" name="${attrs.name}"></div>
-        <div class="form-container tr">
-          <d:invokeBody />
-        </div>
-        <div class="row-set-end row-group-end tr"></div>
-      </j:otherwise>
-    </j:choose>
-  </div>
+  <j:choose>
```

Comment:
not sure exactly what has caused it but extended email looks a bit weird:

Before:

![image](https://user-images.githubusercontent.com/21194782/142163248-bea25184-35ca-4ff3-ad80-755afb3365e5.png)


After:

![image](https://user-images.githubusercontent.com/21194782/142163213-8602a3b0-669b-41f9-871b-48f6da4ac58c.png)